DataAccessLayer/Crud/CategoryCrud.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using DataAccessLayer.DataBaseModels;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataAccessLayer.Crud
+{
+    public class CategoryCrud 
+    {
+        /// <summary>
+        /// Get all categories from table DB
+        /// </summary>
+        /// <returns>List of categories</returns>
+        public List<Category> GetAll()
+        {
+            try
+            {
+                using (LanguageSkillsDBContext db = new LanguageSkillsDBContext())
+                {
+                    return db.Categories.ToList();
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine($"\nClassName - {this.GetType().Name}\nMethodName - {MethodBase.GetCurrentMethod().Name}" +
```

Comment:
This is great that you are using reflection, but you have a lot of duplicated lines all over the code - you maybe can create 1 common method that receives class and method names + Exception and use it everywhere in code.